docker/server.Dockerfile.dev

```diff
@@ -5,8 +5,11 @@ RUN apt-get update && apt-get install -y postgresql-client freetds-dev libaio1 w
 # Install Instantclient Basic Light Oracle and Dependencies
 WORKDIR /opt/oracle
 RUN wget https://download.oracle.com/otn_software/linux/instantclient/instantclient-basiclite-linuxx64.zip && \
+    wget https://tooljet-plugins-production.s3.us-east-2.amazonaws.com/marketplace-assets/oracledb/instantclients/instantclient-basiclite-linux.x64-11.2.0.4.0.zip && \
     unzip instantclient-basiclite-linuxx64.zip && rm -f instantclient-basiclite-linuxx64.zip && \
-    cd /opt/oracle/instantclient* && rm -f *jdbc* *occi* *mysql* *mql1* *ipc1* *jar uidrvci genezi adrci && \
+    unzip instantclient-basiclite-linux.x64-11.2.0.4.0.zip && rm -f iinstantclient-basiclite-linux.x64-11.2.0.4.0.zip && \
```

Comment:
`rm -f iinstantclient-basiclite-linux.x64-11.2.0.4.0.zip` : remove command instant spelling is wrong